next.config.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   reactStrictMode: true,
   output: 'export',
-  basePath: "",
+  basePath: process.env.NODE_ENV == "development" ? "" : "/kata-dashboard-next",
+  // basePath: process.env.NEXT_PUBLIC_BASE_PATH || "",
```

Comment:
This goes hand in hand with the deploy script changes

```suggestion
  basePath: process.env.NEXT_PUBLIC_BASE_PATH || "",
```